Core/Core/People/User.swift

```diff
@@ -54,12 +54,11 @@ extension User: WriteableModel {
 
 extension User {
     public static func displayName(_ name: String, pronouns: String?) -> String {
-        return [
-            name,
-            pronouns.flatMap { "(\($0))" },
-        ]
-        .compactMap { $0 }
-        .joined(separator: " ")
+        if let pronouns = pronouns {
+            let format = NSLocalizedString("%@ (%@)", comment: "Name and pronouns - John (He/Him)")
```

Comment:
thank you for putting in a comment! This probably needs a `bundle: .core`, and I would consider using an id rather than the English string in case we do this same formatting for other things like grade (score) or something.